test/support/session_helpers.rb

```diff
@@ -0,0 +1,8 @@
+module SessionHelpers
+  def mock_logged_in_session
+    request.headers.merge({
+      "Vary" => AccountConcern::ACCOUNT_SESSION_HEADER_NAME,
```

Comment:
This request header isn't needed

Reply:
Cool, Vary is gone and we've simplifed this